pkgs/native_toolchain_c/CHANGELOG.md

```diff
@@ -3,6 +3,8 @@
 - Added `defines` for specifying custom defines.
 - Added `buildModeDefine` to toggle define for current build mode.
 - Added `ndebugDefine` to toggle define of `NDEBUG` for non-debug builds.
+- Generate position independent code for libraries by default and add
```

Comment:
Auto-publishing expects us to publish the first commit with a non `-wip` suffix. So this has to go in 0.2.2.

Reply:
Ah, sorry, when I looked this morning 0.2.1 was not published yet.